### DISCUSSION TITLE
[Docs]: Expanding Docs to include examples used in the website (like Tables)

### DISCUSSION THREAD

**Discussion by SushritPasupuleti** [tamagui/tamagui#249] [Ideas]
**Is your feature request related to a problem? Please describe.**
The recently added compiler section consists of a well-implemented Table component ([found here](https://tamagui.dev/docs/core/exports)).

**Describe the solution you'd like**
Snippets from [here](https://github.com/tamagui/tamagui/blob/master/apps/site/components/MDXComponents.tsx) are a good starting point.

**Comment by natew** [comment]
Would be nice! There's also `PropsTable` which is pretty nice. But both ultimately have too many bugs to release currently, they'd need a lot of love to get really right.

If anyone wants to work on this, let me know. I can help out.

**Reply by SushritPasupuleti** [reply]
Cool, I'll try and see if I can make some time, tables is personally an important feature for me and for anyone else wanting to build something major with Tamagui (something I believe is very much do-able). I'm piecing together the table code in my codebase to make it as atomic asspossible, will connect if I make any good progress! Cheers!
